src/UNL/UCBCN/Manager/Controller.php

```diff
@@ -205,6 +206,26 @@ public function getCalendars() {
         return $user->getCalendars();
     }
 
+    public function hasPermission($permission_description, $calendar_id) 
+    {
+        switch ($permission_description) {
```

Comment:
Oh, ok. Well I can flip that over then. Namespaces are really a silly concept you know